package.json

```diff
@@ -131,7 +146,8 @@
     "i18n:extract": "lingui extract --locale en-US",
     "i18n:compile": "yarn i18n:extract && lingui compile",
     "i18n:pseudo": "lingui extract --locale pseudo && lingui compile",
-    "postinstall": "yarn contracts:compile && yarn graphql:generate && yarn i18n:compile",
+    "prepare": "yarn contracts:compile && yarn graphql:generate && yarn i18n:compile",
```

Comment:
will `prepare` fail w/o an existing `node_modules` dir, since `contracts:compile` references dependencies?

Reply:
yes, but it shouldn't be run until after install according to the docs: https://docs.npmjs.com/cli/v7/using-npm/scripts#life-cycle-scripts

Reply:
>Runs on local `npm install` without any arguments

Am I misunderstanding this?

Reply:
testing for myself

Reply:
Okay, seems to work fine. I read those docs as `prepare` running before `yarn` but it definitely doesn't do that. 🤷